src/kakarot/library.cairo

```diff
@@ -125,6 +126,14 @@ namespace Kakarot {
 
         // Start execution
         let ctx = EVMInstructions.run(ctx);
+        let is_reverted = ExecutionContext.is_reverted(self=ctx);
```

Comment:
I am throwing at the end of 'execute_at_address' for now, but probably with discussion from rpc people about maybe we can propogate the error information to the rpc and let them decide how to convey.

Reply:
@Eikix thoughts of how exceptions should be relayed to the RPC?